sdk/monitor/perf-tests/monitor-opentelemetry-exporter/test/spanExport.spec.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { PerfOptionDictionary } from "@azure/test-utils-perf";
+import { MonitorOpenTelemetryExporterTest } from "./monitorOpenTelemetryExporter.spec";
+import { trace, Span, Tracer, context } from "@opentelemetry/api";
+
+type MonitorOpenTelemetryExporterTestOptions = Record<string, unknown>;
+
+export class SpanExportTest extends MonitorOpenTelemetryExporterTest<MonitorOpenTelemetryExporterTestOptions> {
+  public options: PerfOptionDictionary<MonitorOpenTelemetryExporterTestOptions> = {};
+  constructor() {
+    super();
+  }
+
+  async run(): Promise<void> {
+    async function main() {
+      const tracer = trace.getTracer("testTracer");
+      const parentSpan = tracer.startSpan("main");
+      for (let i = 0; i < 1; i += 1) {
```

Comment:
Typo? why having a loop if only executing once?

Reply:
Maybe I'm missing something here, what is createPerfProgram doing in this case?

Reply:
Was using this for easy modification during testing - I'll clean that up to just make a single span.